package.json

```diff
@@ -23,28 +23,40 @@
   "scripts": {
     "start": "start-storybook -p 3010 -c .storybook",
     "lint": "eslint --ignore-path .gitignore . && stylelint --ignore-path .gitignore \"**/*.css\"",
+    "build": "npm run build:clean && npm run build:js && npm run build:css && npm run build:copy",
+    "build:clean": "rimraf dist styles",
```

Comment:
I think [rimraf](https://github.com/isaacs/rimraf) is not installed.

Reply:
you're right, for some reason I didn't notice any warning about it during builds.